server/http/testhttp/oauth2_complete_test.go

```diff
@@ -55,51 +53,59 @@ func TestOAuth2Complete(t *testing.T) {
 		handler               shttp.Handler
 		r                     *http.Request
 		w                     *mockResponseWriter
+		mockKVStore           *mock_kvstore.MockKVStore
+		mockOAuth2StateStore  *mock_user.MockOAuth2StateStore
+		mockBotPoster         *mock_utils.MockBotPoster
```

Comment:
Since you pass these into the `setupMocks` fn, you don't need to store each of these in each test case -- just create them in the body of the test (right before 241). 

Basically you only need to store things in the test case that are going to be different between test cases. So can you think of anything else that you're storing (and repeating for every test case) that you don't need to?

Reply:
Ok. That makes sense. Definitely no point in storing that in the struct since those never change.